deployment/common/changeset/example/link_transfer.go

```diff
@@ -0,0 +1,182 @@
+package example
+
+import (
+	"fmt"
+	"math/big"
+	"time"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	owner_helpers "github.com/smartcontractkit/ccip-owner-contracts/pkg/gethwrappers"
+	chain_selectors "github.com/smartcontractkit/chain-selectors"
+
+	"github.com/smartcontractkit/ccip-owner-contracts/pkg/proposal/mcms"
+	"github.com/smartcontractkit/ccip-owner-contracts/pkg/proposal/timelock"
+
+	"github.com/smartcontractkit/chainlink/deployment"
+	"github.com/smartcontractkit/chainlink/deployment/common/changeset"
+	"github.com/smartcontractkit/chainlink/deployment/common/proposalutils"
+	"github.com/smartcontractkit/chainlink/deployment/common/types"
+)
+
+const MaxTimelockDelay = 24 * 7 * time.Hour
+
+type TransferConfig struct {
+	To    common.Address
+	Value *big.Int
+}
+
+type MCMSConfig struct {
+	ValidUntil   uint32        // unix time until the proposal will be valid
+	MinDelay     time.Duration // delay for timelock worker to execute the transfers.
+	OverrideRoot bool
+}
+
+type LinkTransferConfig struct {
+	Transfers  map[uint64][]TransferConfig
+	From       common.Address
+	McmsConfig *MCMSConfig
+}
+
+var _ deployment.ChangeSet[*LinkTransferConfig] = LinkTransfer
+
+// Validate checks that the LinkTransferConfig is valid.
+func (cfg LinkTransferConfig) Validate(e deployment.Environment) error {
+	ctx := e.GetContext()
+	// Check that Transfers map has at least one chainSel
+	if len(cfg.Transfers) == 0 {
+		return fmt.Errorf("transfers map must have at least one chainSel")
+	}
+
+	// Check transfers config values.
+	for chainSel, transfers := range cfg.Transfers {
+		_, err := chain_selectors.GetSelectorFamily(chainSel)
+		if err != nil {
+			return fmt.Errorf("invalid chain selector: %w", err)
+		}
+		chain := e.Chains[chainSel]
+		addrs, err := e.ExistingAddresses.AddressesForChain(chainSel)
```

Comment:
The `err` is not checked?

Reply:
thanks fixed here https://github.com/smartcontractkit/chainlink/pull/15512/commits/7dd0fddaf73fa65fb61ca5217b41657b4baa49f6